src/pages/portfolio/index.tsx

```diff
@@ -99,28 +108,57 @@ const useStyles = makeStyles((theme) => ({
   },
   title: {
     marginBottom: theme.spacing(4),
+    width: '100%',
+    zIndex: 1,
 
     [theme.breakpoints.up('sm')]: {
+      position: 'absolute',
       marginBottom: theme.spacing(3),
     },
 
     [theme.breakpoints.up('lg')]: {
+      position: 'absolute',
       margin: theme.spacing(0, 2, 4, 2),
     },
   },
+  navbarTitles: {
+    position: 'relative',
+    zIndex: 2,
+    margin: theme.spacing(-1.2, 2, 4, 2),
+  },
 }));
 
 const PortfolioPage: FC<{ data: ProjectGQL }> = ({ data }) => {
-  const projectData = data.portfolioPage.frontmatter;
   const classes = useStyles();
-  const { componentType, isDesktop } = useComponentType();
+  const defaultLabel = 'All';
+  const projectData = data.portfolioPage.frontmatter;
+  const projectLabels = projectData.projects;
```

Comment:
Could be destructured and renamed 😄 
I would name it `projects`, because `projectLabels` are nested under projects, WDYT? 😄 
```javascript
const { projects } = projectData;
```